account_invoice_merge/__openerp__.py

```diff
@@ -5,7 +5,7 @@
 
 {
     'name': 'Account Invoice Merge Wizard',
-    'version': '9.0.1.0.1',
+    'version': '9.0.1.0.2',
     'category': 'Finance',
     'author': "Elico Corp, Odoo Community Association (OCA)",
     'website': 'http://www.openerp.net.cn',
```

Comment:
please update to https://github.com/OCA/account-invoicing/ as we are at it :)

Reply:
@elicoidal, what change do you mean? I do not understand you.

Reply:
@igallart Change website key value from `http://www.openerp.net.cn` to `https://github.com/OCA/account-invoicing`